apple/internal/apple_framework_import.bzl

```diff
@@ -75,57 +75,18 @@ def _validate_single_framework(framework_paths):
             ),
         )
 
-def _apple_framework_import_impl(ctx):
+def _framework_dirs(framework_imports):
     """Implementation for the apple_framework_import rule."""
-    framework_imports = ctx.files.framework_imports
     framework_groups = group_files_by_directory(
         framework_imports,
         ["framework"],
         attr = "framework_imports",
     )
-
     _validate_single_framework(framework_groups.keys())
```

Comment:
when you rebase, this line was removed

Reply:
@sergiocampama does this validation need to be gone for both static and dynamic frameworks? Or was it one or the other depending on this behavior? Currently https://github.com/bazelbuild/rules_apple/pull/266/ works by finding the binary based on the framework name so that wouldn't work if there was `Foo.framework/Foo` and `Foo.framework/Bar`

Reply:
it's not that there are multiple binaries per framework, it's that there are cases when clients do `framework_imports = glob(["A.framework/**", "B.framework/**"])`, so instead of getting the binary out of 1 framework, you would have to iterate over `framework_groups` getting all the files that match `^.*(framework_name).framework/(framework_name)$` so that `A.framework/A` and `B.framework/B` are added as forced loads

Reply:
Ah right, should be reasonable

Reply:
04d67e8a68f7a9903377d39e3b0b0fa472d2fee5 updates the `alwayslink` support to handle multiple frameworks.